test/test-server/pipeline_handlers.go

```diff
@@ -47,6 +47,12 @@ func handlePipeline(t *testing.T) http.HandlerFunc {
 					UpdatedAt: &UpdatedAt,
 				},
 			}
+
+			// if request to use SR cluster
+			if id == "pipe-11001" {
+				pipeline.Spec.StreamGovernanceCluster = &streamdesignerv1.ObjectReference{Id: "lsrc-1234"}
```

Comment:
Nit: I think these clusters normally have 5 digits
```suggestion
				pipeline.Spec.StreamGovernanceCluster = &streamdesignerv1.ObjectReference{Id: "lsrc-12345"}
```